### DISCUSSION TITLE
Faction power missing placeholders

### DISCUSSION THREAD

**Discussion by Piotro0101** [Dans-Plugins/Medieval-Factions#1793] [Ideas] [answered]
Hello.
At beggining I'm sorry for my language.
There are placeholders for player's max power, but there are no max power for faction or formated power/maxpower.
I will be delighted if you will implement that at future, because it perfectly fits into scoreboards.

**Comment by dmccoystephenson** [comment, marked as answer]
I'm open to this, can you please open a work item?
https://github.com/Dans-Plugins/Medieval-Factions/issues/new/choose
